libc/include/assert.h.def

```diff
@@ -6,9 +6,16 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifndef LLVM_LIBC_ASSERT_H
+#define LLVM_LIBC_ASSERT_H
+
 #include "__llvm-libc-common.h"
 
 // This file may be usefully included multiple times to change assert()'s
 // definition based on NDEBUG.
 
 %%public_api()
+
+#include "llvm-libc-macros/assert-macros.h"
```

Comment:
If `assert-macros.h` is ONLY going to have this one macro def, then let's just define `__STDC_VERSION_ASSERT_H__` here and then we have no need for an `assert-macros.h`?  Then the cmake change for that can be undone, too.

Though I guess it's nice that the include test only needs to depend on libc.include.llvm-libc-macros.assert_macros (I'm not sure that the cmake pattern I used for the stdbit tests are as elegant as they could be).  @michaelrj-google and I discussed this a bit yesterday.  @michaelrj-google do you have thoughts here?  There's also `SimpleMacroDef` which we could use in an api.td file to define this.

Reply:
Sure thing! I'll just include that define in `assert.h.def` for now

Reply:
Oop missed the comment above, I'll wait it out then